src/service/models.py

```diff
@@ -432,7 +432,27 @@ class ProcessType(str, Enum):
     """ A seletion process. """
 
 
-class DataProductProcess(ProcessAttributes):
+class DataProductProcessIdentifier(BaseModel):
```

Comment:
Idk if I love this verbiage, but I get it, once I understood it.

Reply:
just a little odd to have the type/data product ID also in there.

DataProductProccessLocator? Idk.

Reply:
yeah, not a huge fan either but it describes what it does and i couldn't think of anything better. If you have better ideas I'm happy to consider it